content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
 
-```bash
-iface enp4s0f2 inet manual
-```
+![nvtop](/images/battery/nvtop.png)
+
+If `nvtop` is showing a lot of GPU activity when no GPU-heavy applications (like games or 3D software) are running, there may be an program which is using the GPU when it's not supposed to be.
 
-This new configuration will take effect after a reboot. To disable the interface immediately, run this command:
+### Intel graphics: `intel_gpu_top`
+
+Integrated Intel graphics have an additional monitoring tool called `intel_gpu_top`. It can be installed from the terminal with this command, along with other tools for Intel graphics:
 
 ```bash
-sudo ifconfig enp4s0f2 down
+sudo apt install intel-gpu-tools
 ```
 
-Then, if Ethernet needs to be used, this command can be run to enable it for the session:
+and then run with the command:
 
 ```bash
-sudo ifconfig enp4s0f2 up
+sudo intel_gpu_top
 ```
 
-## Useful Commands
+This screenshot shows `intel_gpu_top` on a system running some normal applications with a 3D accelerated desktop, along with a video player that's using the Intel hardware for decoding.
+
+![intel_gpu_top](/images/battery/intel_gpu_top.png)
+
+If `intel_gpu_top` is showing a lot of GPU activity when no GPU-heavy applications (like games or 3D software) are running, there may be an program which is using the GPU when it's not supposed to be.
+
+## Checking battery health
 
 This command will show the information that your computer can read about the battery:
 
 ```bash
 upower -d
 ```
 
-The output from `upower -d` is also included when generating log files in Settings -> Support -> Create Log Files:
+The "Capacity" statistic compares the original power capacity that the battery had when it with new, with the current maximum power capacity. Batteries do lose maximum power charge as they age, and this can be a good way to see if the battery needs replacement.
 
-![Create Log Files](/images/battery/create-logs.png)
+## Inaccurate battery percentages
 
-If the battery life indicator is inaccurate, this will remove the stored statistics:
+If the battery life indicator at the top of the System Menu is inaccurate, which can often happen on a new system or after a battery has been replaced, this command will remove the stored battery statistics:
 
 ```bash
 sudo rm /var/lib/upower/*
 ```
 
-After a few charge/discharge cycles, the indicator should be more accurate.
+After a few charge/discharge cycles, the indicator should become more accurate as the system learns how to estimate the correct percentage.
+
+## Tools we do not recommend
+
+The [TLP power maagement tool for Linux](https://linrunner.de/tlp/index.html) is popular on other Linx distributions, but it is not compatible with `system76-power`, and both System76 and [the TLP developers](https://linrunner.de/tlp/faq/installation.html#does-tlp-conflict-with-other-power-management-tools) do not recommend installing TLP on Pop!\_OS.
```

Comment:
Typo in "management" and "Linux" spelling

Reply:
Done, will be updated on the next git push. (Man, I missed a lot of typos at the end of the day yesterday.)